pyflakesbears/NoFutureImportBear.py

```diff
@@ -1,27 +1,82 @@
 from coalib.bears.LocalBear import LocalBear
 from coalib.results.Result import Result
+from coalib.results.Diff import Diff
 from pyflakesbears.PyFlakesASTBear import PyFlakesASTBear
 from pyflakes.checker import FutureImportation
 
 
 class NoFutureImportBear(LocalBear):
     """
-    Uses pyflakes-enhance-AST to detect use of future imports
+    Uses pyflakes-enhance-AST to remove future imports
     """
-    LANGUAGES = {'Python', 'Python 2', 'Python 3'}
+
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     BEAR_DEPS = {PyFlakesASTBear}
 
+    def remove_future_imports(self, file, lineno, corrected_lines):
+        """
+        Removes all ImportFrom nodes from the input line
+        """
+        def handle_backslash(line, lineno, diff, corrected_lines):
```

Comment:
while this is acceptable for an interim solution, it is PyFlakesASTBear which needs to provide the helpers needed to make backslash and multi-statement lines less troublesome to deal with.

We dont want every bear to have to write custom code for these very real problems which could occur on every single line.